content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -88,24 +95,29 @@ COMPOSE_PROJECT_NAME=db-compose
 tree . -a
 .
 ├── .env
-├── docker-compose.yml
+├── docker-compose.[yml / yaml]
 └── init
     └── 1.create-user-table.sql
 ```
 
 #  Useful Docker compose commands
-- To launch a container.
+- To launch a container
```

Comment:
Ah, just a phrase, not a sentence right?

Reply:
Yes, that's right!